astroquery/eso/core.py

```diff
@@ -769,9 +915,11 @@ def get_associated_files(self, datasets: List[str], *, mode: str = "raw",
         # remove input datasets from calselector results
         return list(associated_files.difference(set(datasets)))
 
-    @deprecated_renamed_argument(('request_all_objects', 'request_id'), (None, None), since=['0.4.7', '0.4.7'])
-    def retrieve_data(self, datasets, *, continuation=False, destination=None, with_calib=None,
-                      request_all_objects=False, unzip=True, request_id=None):
+    @deprecated_renamed_argument(('request_all_objects', 'request_id'), (None, None),
+                                 since=['0.4.7', '0.4.7'])
```

Comment:
This is a bit unrelated so no need to do it in this PR, but 0.4.7 was a while ago, so these deprecations can be now removed.